2022-05-23 13:08:40,685 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:08:40,686 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:08:40,686 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:08:40,686 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:08:40,836 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:08:40,837 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:08:40,967 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:08:40,968 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:08:46,437 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 13:14:13,125 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:14:13,127 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:14:13,127 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:14:13,127 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:14:13,302 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:14:13,303 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:14:13,428 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:14:13,428 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:17:21,551 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 13:17:23,203 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:17:23,204 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:17:23,205 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:17:23,205 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:17:23,334 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:17:23,335 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:17:23,411 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:17:23,411 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:17:33,724 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/channels/6557701/messages, error: 400, 返回内容: {"message":"400: Bad Request","code":10001}, trace_id:528faaa5f52d81b37b3868b3daf335c4
2022-05-23 13:18:17,872 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/channels/6557701/messages, error: 400, 返回内容: {"message":"400: Bad Request","code":10001}, trace_id:cb68d66bfdaa71c48a05b831aabd2aa9
2022-05-23 13:19:13,818 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 13:19:15,919 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:19:15,920 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:19:15,921 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:19:15,921 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:19:16,068 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:19:16,068 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:19:16,174 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:19:16,174 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:19:24,541 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/channels/6557701/messages, error: 400, 返回内容: {"message":"400: Bad Request","code":10001}, trace_id:25abbd9029e3c9b6836fb9f54af1053a
2022-05-23 13:22:11,791 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 13:22:14,187 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:22:14,191 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:22:14,192 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:22:14,192 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:22:14,363 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:22:14,364 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:22:14,483 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:22:14,484 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:22:59,351 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1007206365482934096/messages, error: 500, 返回内容: {"code":304049,"message":"push dm message reach limit"}, trace_id:19a6233dbb69d32ce4fee6f1336aad1a
2022-05-23 13:32:39,101 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 13:32:40,584 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:32:40,585 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:32:40,585 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:32:40,585 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:32:40,722 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:32:40,723 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:32:40,811 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:32:40,812 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:33:58,476 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 13:34:00,365 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:34:00,367 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:34:00,367 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:34:00,367 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:34:00,521 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:34:00,522 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:34:00,647 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:34:00,647 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:34:31,394 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1007206365482934096/messages, error: 500, 返回内容: {"code":304049,"message":"push dm message reach limit"}, trace_id:f834535756bae3d91abd70a17f062285
2022-05-23 13:34:49,441 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 13:44:52,138 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 13:44:52,139 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 13:44:52,140 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 13:44:52,140 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 13:44:52,287 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 13:44:52,288 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 13:44:52,421 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 13:44:52,421 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 13:47:01,134 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 14:00:10,217 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 14:00:10,218 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 14:00:10,219 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 14:00:10,219 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 14:00:10,360 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 14:00:10,361 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 14:00:10,506 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 14:00:10,506 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 14:00:12,043 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 14:12:08,753 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:c5e726897bb7215ade29e80c4323e961
2022-05-23 14:12:24,670 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:ea531d294c0a3956d5e4dc0715d5d236
2022-05-23 14:12:55,129 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:29c33ad3818e2f6fce10096eb46bc4df
2022-05-23 14:14:16,188 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:559238eabff51e40cef9d9f4740d3776
2022-05-23 14:15:13,174 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:9bc11fc4ecc910059a62f366f2a67176
2022-05-23 14:15:38,873 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:4df2e58b9002acb1b22ed4bb7085ab7e
2022-05-23 14:16:25,852 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:fcb6da309642c18e37b8ffbec172fa98
2022-05-23 14:16:48,705 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:3174347d2b5ae26ecaf64d88f35f1414
2022-05-23 14:17:14,328 [ERROR] _handle_response (http.py:57): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/gateway/bot, error: 401, 返回内容: b'{"message":"wrong bot token","code":11242}', trace_id:a9e02371b725d2818a08103180954171
2022-05-23 14:17:32,558 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 14:17:32,558 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 14:17:32,559 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 14:17:32,559 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 14:17:32,705 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 14:17:32,706 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 14:17:32,823 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 14:17:32,824 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 14:17:45,561 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1007206365482934096/messages, error: 500, 返回内容: {"code":304049,"message":"push dm message reach limit"}, trace_id:7277aa7555c1797a39fe37a288ad0177
2022-05-23 14:18:53,750 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1007206365482934096/messages, error: 500, 返回内容: {"code":304049,"message":"push dm message reach limit"}, trace_id:a5fd49f32b601ae2fd9c68c66e85914c
2022-05-23 14:19:03,691 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 14:21:23,852 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 14:21:23,854 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 14:21:23,854 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 14:21:23,854 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 14:21:24,019 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 14:21:24,020 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 14:21:24,118 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 14:21:24,119 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 14:21:34,662 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1007206365482934096/messages, error: 500, 返回内容: {"code":304049,"message":"push dm message reach limit"}, trace_id:13b388273b2ff6aaffa8dc03b4556783
2022-05-23 14:21:54,626 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 20:00:21,212 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 20:00:21,214 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 20:00:21,215 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 20:00:21,215 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 20:00:21,386 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 20:00:21,386 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 20:00:21,517 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 20:00:21,517 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 20:00:23,554 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 20:11:26,443 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 20:11:26,446 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 20:11:26,446 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 20:11:26,447 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 20:11:26,603 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 20:11:26,604 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 20:11:26,705 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 20:11:26,705 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 20:11:28,341 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 20:53:43,734 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 20:53:43,737 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 20:53:43,737 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 20:53:43,737 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 20:53:43,897 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 20:53:43,898 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 20:53:43,980 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 20:53:43,980 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 20:55:13,467 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 20:55:16,651 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 20:55:16,655 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 20:55:16,656 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 20:55:16,656 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 20:55:16,815 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 20:55:16,817 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 20:55:16,945 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 20:55:16,945 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 20:57:01,055 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 20:57:03,449 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 20:57:03,453 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 20:57:03,454 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 20:57:03,454 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 20:57:03,612 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 20:57:03,614 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 20:57:03,691 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 20:57:03,692 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 20:58:00,085 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 20:58:34,087 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 20:58:34,088 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 20:58:34,088 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 20:58:34,088 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 20:58:34,251 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 20:58:34,252 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 20:58:34,388 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 20:58:34,389 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:00:14,665 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:00:16,374 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:00:16,375 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:00:16,375 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:00:16,375 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:00:16,529 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:00:16,530 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:00:16,610 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:00:16,610 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:01:00,462 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:01:03,265 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:01:03,268 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:01:03,269 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:01:03,269 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:01:03,448 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:01:03,449 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:01:03,553 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:01:03,554 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:01:25,855 [INFO] start_session (ws_async_manager.py:95): [连接管理]服务强行停止!
2022-05-23 21:01:48,793 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:01:48,793 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:01:48,794 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:01:48,794 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:01:48,940 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:01:48,940 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:01:49,088 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:01:49,088 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:02:11,489 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:02:13,071 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:02:13,072 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:02:13,072 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:02:13,072 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:02:13,268 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:02:13,268 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:02:13,399 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:02:13,399 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:04:42,479 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:04:44,286 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:04:44,288 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:04:44,288 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:04:44,288 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:04:44,440 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:04:44,441 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:04:44,562 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:04:44,562 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:05:36,952 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:05:39,977 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:05:39,981 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:05:39,982 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:05:39,982 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:05:40,836 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:05:40,838 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:05:41,037 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:05:41,037 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:06:29,159 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:06:30,953 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:06:30,954 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:06:30,954 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:06:30,954 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:06:31,128 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:06:31,129 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:06:31,214 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:06:31,214 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:07:11,494 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:07:12,948 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:07:12,949 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:07:12,949 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:07:12,949 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:07:13,086 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:07:13,086 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:07:13,207 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:07:13,207 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:07:37,432 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:07:40,493 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:07:40,497 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:07:40,498 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:07:40,498 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:07:40,667 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:07:40,669 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:07:40,806 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:07:40,806 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:10:23,662 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:10:25,787 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:10:25,788 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:10:25,788 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:10:25,788 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:10:25,961 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:10:25,962 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:10:26,084 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:10:26,084 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:10:39,383 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:10:42,238 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:10:42,242 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:10:42,243 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:10:42,243 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:10:42,411 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:10:42,413 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:10:42,541 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:10:42,541 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:14:40,141 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:14:42,348 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:14:42,349 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:14:42,349 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:14:42,349 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:14:42,497 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:14:42,497 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:14:42,635 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:14:42,635 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:15:13,306 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:17:29,635 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:17:29,637 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:17:29,637 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:17:29,637 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:17:29,797 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:17:29,797 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:17:29,920 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:17:29,920 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:17:38,343 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/channels/6557701/messages, error: 400, 返回内容: {"code":50006,"message":"empty msg"}, trace_id:c4b16509091bcf28ed8e86a683f689ee
2022-05-23 21:18:57,948 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:18:59,321 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:18:59,322 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:18:59,323 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:18:59,323 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:18:59,479 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:18:59,479 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:18:59,600 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:18:59,600 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:19:05,865 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/channels/6557701/messages, error: 400, 返回内容: {"code":50006,"message":"empty msg"}, trace_id:d48128612e0edadb38ac90c8c59e5e1c
2022-05-23 21:19:51,852 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:19:53,847 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:19:53,848 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:19:53,848 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:19:53,848 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:19:54,078 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:19:54,080 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:19:54,239 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:19:54,239 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:20:00,594 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/channels/6557701/messages, error: 400, 返回内容: {"code":50006,"message":"empty msg"}, trace_id:63bb55b26075df80075183eb0e91a962
2022-05-23 21:20:08,867 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:20:10,299 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:20:10,300 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:20:10,300 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:20:10,300 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:20:10,470 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:20:10,470 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:20:10,564 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:20:10,565 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:20:57,021 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:20:59,551 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:20:59,555 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:20:59,556 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:20:59,556 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:20:59,720 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:20:59,722 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:20:59,867 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:20:59,867 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:21:19,884 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/channels/6557701/messages, error: 400, 返回内容: {"code":50006,"message":"empty msg"}, trace_id:a399f4c5cbb610a6ab56e24e0f4ca053
2022-05-23 21:45:30,399 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 21:45:33,246 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 21:45:33,250 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 21:45:33,251 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 21:45:33,251 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 21:45:33,456 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 21:45:33,457 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 21:45:33,632 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 21:45:33,633 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 21:47:06,899 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 23:41:16,256 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 23:41:16,262 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 23:41:16,263 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 23:41:16,264 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 23:41:16,667 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 23:41:16,668 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 23:41:16,757 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 23:41:16,758 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 23:43:28,440 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 23:43:30,953 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 23:43:30,957 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 23:43:30,958 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 23:43:30,958 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 23:43:31,096 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 23:43:31,097 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 23:43:31,219 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 23:43:31,220 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 23:45:35,055 [INFO] start_session (ws_async_manager.py:95): [连接管理]服务强行停止!
2022-05-23 23:45:37,551 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 23:45:37,556 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 23:45:37,556 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 23:45:37,557 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 23:45:37,735 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 23:45:37,737 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 23:45:37,859 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 23:45:37,859 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 23:46:47,291 [INFO] start_session (ws_async_manager.py:95): [连接管理]服务强行停止!
2022-05-23 23:46:49,601 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 23:46:49,605 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 23:46:49,606 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 23:46:49,606 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 23:46:49,754 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 23:46:49,756 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 23:46:49,877 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 23:46:49,877 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 23:47:55,468 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 23:47:57,660 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 23:47:57,664 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 23:47:57,665 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 23:47:57,665 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 23:47:57,825 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 23:47:57,827 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 23:47:57,952 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 23:47:57,952 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 23:53:27,463 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
2022-05-23 23:53:29,786 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 23:53:29,790 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 23:53:29,791 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 23:53:29,791 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 23:53:29,947 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 23:53:29,948 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 23:53:30,081 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 23:53:30,081 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 23:54:08,312 [INFO] start (ws_async_manager.py:54): [连接管理]程序启动...
2022-05-23 23:54:08,313 [INFO] run (ws_async_pool.py:34): [连接池]最大并发连接数: 1, 启动会话数: 1
2022-05-23 23:54:08,313 [INFO] new_connect (ws_async_manager.py:110): [连接管理]新会话启动中...
2022-05-23 23:54:08,313 [INFO] connect (ws_async_client.py:79): [ws连接]启动中...
2022-05-23 23:54:08,448 [INFO] identify (ws_async_client.py:113): [ws连接]鉴权中...
2022-05-23 23:54:08,448 [INFO] _send_heartbeat (ws_async_client.py:193): [ws连接]心跳检测启动...
2022-05-23 23:54:08,547 [INFO] on_message (ws_async_client.py:57): [ws连接]鉴权成功
2022-05-23 23:54:08,547 [INFO] on_message (ws_async_client.py:62): [ws连接]程序启动成功！
2022-05-23 23:54:16,094 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1007206365482934096/messages, error: 500, 返回内容: {"code":304049,"message":"push dm message reach limit"}, trace_id:8f5330efb6237f859a11662479034eb2
2022-05-23 23:54:42,194 [ERROR] _handle_response (async_http.py:61): [HTTP]接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1007206365482934096/messages, error: 500, 返回内容: {"code":304049,"message":"push dm message reach limit"}, trace_id:511eeec2952cbe73ebad9d4e14282940
2022-05-23 23:55:50,410 [INFO] start_session (ws_async_manager.py:98): [连接管理]服务强行停止!
